datastore/mongo/event.go

```diff
@@ -164,18 +163,23 @@ func (db *eventRepo) FindEventByID(ctx context.Context, id string) (*datastore.E
 }
 
 func (db *eventRepo) FindEventsByIDs(ctx context.Context, ids []string) ([]datastore.Event, error) {
-	m := make([]datastore.Event, 0)
+	ctx = db.setCollectionInContext(ctx)
+
+	var event []datastore.Event
 
-	filter := bson.M{"uid": bson.M{"$in": ids}, "document_status": datastore.ActiveDocumentStatus}
+	filter := bson.M{"uid": bson.M{"$in": ids}}
```

Comment:
If we remove the document status filter, won't it also return deleted records?

Reply:
Yeah, similar issue.